### ISSUE TITLE
Views of saved searches do not include time-related views

### ISSUE THREAD

**Issue by HughCraig** [HughCraig/GHAP#317]
If you save a search the views available do not include Journey Times, Timeline, or Werekata Flyover By Time. This makes sense with searches including multiple layers, some of which may not include valid temporal data. However, in the case of a saved search within a layer which does include valid temporal data, it would be advantageous to have the option to create the three time-related views.

**Comment by IanMcCrabb** [comment]
We could provide these views by default as long as they fail gracefully when no time data is present.

**Comment by BillPascoe** [comment]
Yes, just noticed this too and was going to make an issue. This should be a high priority. I'm having trouble making good examples of faceted search without this.

**Comment by BillPascoe** [comment]
Some useful layers for testing this might be:
https://ghap.tlcmap.org/layers/384
https://ghap.tlcmap.org/layers/379
https://ghap.tlcmap.org/layers/1262
